.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,70 @@
+<!--  All Submissions -->
+
+
+## Community Note
+<!-- Please leave the community note as is. -->
+* Please vote on this PR by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original PR to help the community and maintainers prioritize for review
+* Please do not leave "+1" or "me too" comments, they generate extra noise for PR followers and do not help prioritize for review
+
+
+## PR Checklist
+
+- [ ] Have you followed the guidelines in our [Contributing Documentation](../contributing/README.md)?
+- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+- [ ] Have you used a meaningful PR description to help maintainers and other users understand this change and help prevent duplicate work?
```

Comment:
this is referring to the title?
```suggestion
- [ ] Have you used a meaningful PR title to help maintainers and other users understand this change and help prevent duplicate work?
```

Reply:
I'm not sure. It could really be in reference to either. @jackofallops Do you know if this is a typo or is supposed to be description?

Reply:
I think it makes more sense to be Title then description given the example is a title